src/sign.c

```diff
@@ -22,88 +22,87 @@ typedef struct sign sign_T;
 
 struct sign
 {
-    sign_T	*sn_next;	// next sign in list
-    int		sn_typenr;	// type number of sign
-    char_u	*sn_name;	// name of sign
-    char_u	*sn_icon;	// name of pixmap
-# ifdef FEAT_SIGN_ICONS
-    void	*sn_image;	// icon image
-# endif
-    char_u	*sn_text;	// text used instead of pixmap
-    int		sn_line_hl;	// highlight ID for line
-    int		sn_text_hl;	// highlight ID for text
-    int		sn_cul_hl;	// highlight ID for text on current line when 'cursorline' is set
-    int		sn_num_hl;	// highlight ID for line number
-    int		sn_priority;	// default priority of this sign, -1 means SIGN_DEF_PRIO
+    sign_T *sn_next; // next sign in list
+    int sn_typenr;   // type number of sign
+    char_u *sn_name; // name of sign
+    char_u *sn_icon; // name of pixmap
+#ifdef FEAT_SIGN_ICONS
+    void *sn_image;  // icon image
+#endif
+    char_u *sn_text; // text used instead of pixmap
+    int sn_line_hl;  // highlight ID for line
+    int sn_text_hl;  // highlight ID for text
+    int sn_cul_hl; // highlight ID for text on current line when 'cursorline' is set
+    int sn_num_hl;   // highlight ID for line number
+    int sn_priority; // default priority of this sign, -1 means SIGN_DEF_PRIO
```

Comment:
The Vim coding convention is to align all the variable names in a structure using 'shiftwidth' (and 'softtabstop').  With this new indentation, the names are not aligned.